ledger/daml-on-sql/README.rst

```diff
@@ -0,0 +1,703 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+DAML-on-SQL
+###########
+
+DAML-on-SQL is a PostgreSQL-based DAML ledger implementation.
+
+Setup PostgreSQL and run
+************************
+
+Before starting, you need to perform the following steps:
+
+- create an initially empty PostgresSQL database that DAML-on-SQL can access
+- have a database user for DAML-on-SQL that has authority to execute DDL operations
+
+This is because DAML-on-SQL manages its own database schema,
+applying migrations if necessary when upgrading versions.
+
+To specify the PostgreSQL instance you wish to connect, use the
+``--sql-backend-jdbcurl <value>`` command line option, where ``<value>``
+is a valid JDBC URL containing the username, password and database
+name to connect to.
+
+Here is an example for such a URL: ``jdbc:postgresql://localhost/test?user=fred&password=secret``
+
+Due to possible conflicts between the ``&`` character and various terminal
+shells, we recommend quoting the JDBC URL like so:
+
+.. code-block:: none
+
+  $ java -jar daml-on-sql-<version>.jar Main.dar --sql-backend-jdbcurl 'jdbc:postgresql://localhost/test?user=fred&password=secret'
```

Comment:
Let's decouple managing the the daml on sql process from the "deployment" of DAR packages.
```suggestion
  $ java -jar daml-on-sql-<version>.jar --sql-backend-jdbcurl 'jdbc:postgresql://localhost/test?user=fred&password=secret'
```